app/models/rules_of_origin/data_set.rb

```diff
@@ -22,6 +24,29 @@ def default_heading_mappings
           importer.import(skip_invalid_rows: true)
         end
       end
+
+      # FIXME: Mocked data - to be removed once data loading is turned on
+
+      def load_mocked
+        new mocked_scheme_set, mocked_rule_set, mocked_mappings
+      end
+
+      def mocked_scheme_set
```

Comment:
it seems mocked_scheme_set, mocked_rule_set and mocked_mappings should be private methods

Reply:
These are private? just in the class' namespace?